### PR TITLE
2023 import

### LINKED ISSUE #203: Update to 2023
- WPILib 2023.1.1
- RevLib 2023.1.1 (https://software-metadata.revrobotics.com/REVLib-2023.json)
- CTRE 5.30.2 (https://maven.ctr-electronics.com/release/com/ctre/phoenix/Phoenix5-frc2023-latest.json)
- PathPlannerLib 2023.2.3 (https://3015rangerrobotics.github.io/pathplannerlib/PathplannerLib.json)

blocks frc5572/Jetsi-Offseason#21

### PR DESCRIPTION
**Pull request by agrinmanriv0537**
Applying 2023 dependency changes to 2022 code, everything functional. Closes #203

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -1,11 +0,0 @@
...
-    "recommendations": [
-        "vscjava.vscode-java-debug",
-        "editorconfig.editorconfig",
-        "vscjava.vscode-java-pack",
-        "github.vscode-pull-request-github",
-        "eamodio.gitlens",
-        "redhat.java",
-        "wpilibsuite.vscode-wpilib"
-    ]
-}
```
Comment (legoguy1000) [inline .vscode/extensions.json]: don't delete

File: .vscode/settings.json [hunk 2]
```diff
@@ -18,25 +18,12 @@
...
-  "java.format.settings.profile": "GoogleStyle",
-  "[java]": {
-    "editor.defaultFormatter": "redhat.java",
-  },
-  "editor.codeActionsOnSave": {
-    "source.organizeImports": true
-  },
-  "editor.formatOnSave": true,
-  "editor.formatOnPaste": true,
-  "terminal.integrated.defaultProfile.windows": "Git Bash"
```
Comment (legoguy1000) [inline .vscode/settings.json]: don't delete.  THis is just from the import.

File: src/main/java/frc/lib/InfiniteCommand.java [hunk 3]
```diff
@@ -1,30 +1,30 @@
-package frc.lib;
+// package frc.lib;
```
Comment (legoguy1000) [inline src/main/java/frc/lib/InfiniteCommand.java]: If we're not using this, just delete it.

File: src/main/java/frc/robot/autos/P1_3B.java [hunk 4]
```diff
@@ -55,41 +55,41 @@ public P1_3B(Swerve swerve, Shooter shooter, InnerMagazine innerMagazine,
...
-                })).andThen(new FeedShooter(this.innerMagazine, this.outerMagazine, this.shooter,
-                    this.intake).withTimeout(1.5));
+            new SequentialCommandGroup(autoDrive, new ZeroMotorsWaitCommand(swerve, .5));
+                // .deadlineWith(new StartEndCommand(() -> {
+                //     intake.intakeDeploy();
+                //     outerMagazine.magazineUp();
+                // }, () -> {
+                //     outerMagazine.magazineStop();
+                // })).andThen(new FeedShooter(this.innerMagazine, this.outerMagazine, this.shooter,
+                //     this.intake).withTimeout(1.5));
```
Comment (legoguy1000) [inline src/main/java/frc/robot/autos/P1_3B.java]: uncomment this so its back to what it was.

File: src/main/java/frc/robot/autos/P1_3B.java [hunk 5]
```diff
@@ -55,41 +55,41 @@ public P1_3B(Swerve swerve, Shooter shooter, InnerMagazine innerMagazine,
...
+                    }), new InnerMagIntake(this.innerMagazine))
             .andThen(
                 new FeedShooter(this.innerMagazine, this.outerMagazine, this.shooter, this.intake)
                     .withTimeout(.7));
 
         addCommands(new InstantCommand(() -> swerve.zeroGyro()),
             new InstantCommand(
                 () -> swerve.resetOdometry(new Pose2d(initialState.poseMeters.getTranslation(),
-                    initialState.holonomicRotation))),
+                    initialState.holonomicRotation))), part1, part2,
```
Comment (legoguy1000) [inline src/main/java/frc/robot/autos/P1_3B.java]: put this back to what it was

File: src/main/java/frc/robot/modules/swervedrive/SwerveModule.java [hunk 6]
```diff
@@ -45,6 +47,8 @@ public SwerveModule(int moduleNumber, SwerveModuleConstants moduleConstants) {
         /* Drive Motor Config */
         driveMotor = new TalonFX(moduleConstants.driveMotorID, "canivore");
         configDriveMotor();
+        // angleMotor.setSelectedSensorPosition(0);
+
```
Comment (legoguy1000) [inline src/main/java/frc/robot/modules/swervedrive/SwerveModule.java]: delete

File: src/main/java/frc/robot/modules/swervedrive/SwerveModule.java [hunk 7]
```diff
@@ -108,6 +112,25 @@ public void setTurnAngle(double rotationSpeed) {
...
+        // double velocity = Conversions.falconToMPS(driveMotor.getSelectedSensorVelocity(),
+        // Constants.Swerve.wheelCircumference, Constants.Swerve.driveGearRatio);
+        double position = Conversions.falconToMeters(driveMotor.getSelectedSensorPosition(),
+            Constants.Swerve.driveGearRatio, Constants.Swerve.wheelCircumference);
+
+
+
+        Rotation2d angle = Rotation2d.fromDegrees(Conversions.falconToDegrees(angleMotor.getSelectedSensorPosition(), Constants.Swerve.angleGearRatio));
+        return new SwerveModulePosition(position, angle);
+        // return new SwerveModulePosition(.5, new Rotation2d(.5));
```
Comment (legoguy1000) [inline src/main/java/frc/robot/modules/swervedrive/SwerveModule.java]: delete

**Review by legoguy1000 — CHANGES_REQUESTED** [review]

**Review by gvaldez7206 — DISMISSED** [review]
Looks good

**Review by gvaldez7206 — APPROVED** [review]
Looks good

**Review by legoguy1000 — APPROVED** [review]

**Pull request merged**